abdulfatai360/banka
# #165805723 Implement feedback on UI design

#### What does this PR do?
Implement feedback about UI from panelists on Pitch day

#### Description of Task to be completed?
- Have some UI components redesigned

#### How should this be manually tested?
On Terminal:
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto your local machine
- Navigate to the repo directory by running `cd banka/`
- Ensure your working directory is clean by running  `git status`  command
- Get a copy of the PR by typing  `git fetch origin pull/59/head:ch-modal-signin-signup-165805723`
- Now that you have a copy of the branch, switch to it using  `git checkout ch-modal-signin-signup-165805723`
- Your directory will now be an exact copy of the PR

#### What are the relevant pivotal tracker stories?
[#165805723](https://www.pivotaltracker.com/story/show/165805723)
